src/lib/terminal.js

```diff
@@ -168,25 +168,41 @@ class Terminal {
   }
 
   addText(data){
-    const { lines, pointerLine, pointerColumn, trimCount, maxLines } = this.state;
-    const line = lines[pointerLine] || '';
-    if(pointerColumn < line.length){
-      const start = line.slice(0, pointerColumn);
-      const end = line.slice(pointerColumn);
-      lines[pointerLine] = start + data + end;
-    }else if(pointerColumn > line.length){
-      lines[pointerLine] = _.padRight(line, pointerColumn) + data;
-    }else{
-      lines[pointerLine] = line + data;
+    const { lines, lineWrap, pointerLine, pointerColumn, trimCount, maxLines } = this.state;
+
+    var newPointerColumn = pointerColumn;
+    var newPointerLine = pointerLine;
```

Comment:
why are you defining this as `newPointerLine`? It seems like you can just use `pointerLine` since it only gets incremented if the line wraps